scripts/CEE/collect-verbose-ca-logs/script.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+set -e
+set -o nounset
+set -o pipefail
+
+# Set variables
+GREEN="\033[32m"
+RESET="\033[0m"
+NAMESPACE="openshift-machine-api"
+CA_NAME="default"
+
+# Get current logVerbosity value
+current_log_verbosity=$(oc get ca $CA_NAME -o jsonpath='{.spec.logVerbosity}')
```

Comment:
Can we use the full resource name `clusterautoscalers` here rather than the short version?

Reply:
the same as the other places.